webui/src/pages/repositories/repository/objects.jsx

```diff
@@ -362,6 +375,7 @@ const ReadmeContainer = ({repo, reference, path='', refreshDep=''}) => {
             error={error}
             loading={loading}
             showFullNavigator={false}
+            presign={config.pre_sign_support}
```

Comment:
Need to check pre_sign_support_ui - always

Reply:
we check it on the backend when deciding on the pre_sign_support_ui